docs/content/contribution-guidelines/contributing-inc.md

```diff
@@ -33,6 +33,32 @@ Your assistance in improving documentation is highly valued, regardless of your
 
 To claim an issue that you are interested in, kindly leave a comment on the issue and request the maintainers to assign it to you.
 
+#### GitHub Slash Commands
+
+KubeStellar uses Prow and GitHub bots to help manage issues and pull requests through slash commands. These commands should be written as comments on their own line:
+
+**Issue Management Commands:**
+- `/assign @username` - Assign an issue to a specific user
+- `/unassign @username` - Remove assignment from a user  
+- `/assign` - Assign the issue to yourself
+- `/unassign` - Remove your assignment
+- `/good-first-issue` - Add the "good first issue" label
+- `/help-wanted` - Add the "help wanted" label
```

Comment:
I tested this, at https://github.com/kubestellar/kubestellar/issues/3186#issuecomment-3156314832 . It provoked no reaction. Also note, the documentation at https://prow.k8s.io/command-help says that the proper command for this is `/help`. I also tested `/help`, it also provoked no reaction. @clubanderson, is this breakage in our Prow installation?